.github/workflows/ci_check.yml

```diff
@@ -0,0 +1,35 @@
+name: Check
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event_name }}
```

Comment:
What's this?

Reply:
Cancel existing jobs if new event triggered. For example, the author keep pushing new commits to the same branch.